docs/index.rst

```diff
@@ -2,78 +2,110 @@
    sphinx-quickstart on Mon Feb 17 11:41:38 2020.
    You can adapt this file completely to your liking, but it should at least
    contain the root `toctree` directive.
+******************************
+Flax
+******************************
+.. grid::
 
-Flax documentation
-==================
+   .. grid-item::
+      :columns: 7 5 7 5
+      :margin: 0 2 auto auto 
 
-Flax is a neural network library and ecosystem for JAX that is
-designed for flexibility. Flax is in use by a growing community of
-researchers and engineers at Google who happily use Flax for their
-daily research.
+      .. image:: ./flax.png
```

Comment:
the logo is cool, but it doesn't add anything.
also, we now have it twice on this page (top left, center).

let's instead use the precious screen estate to show links / text / code ...

Reply:
I knew this would be controversial 😅 I took inspiration from the [FastAPI](https://fastapi.tiangolo.com/) landing page, I like that yields a little bit of a dramatic entrance, it looks better in this other theme where the nav bar is at the top and the icon is small so it doesn't look repetitive:

![Screenshot 2022-06-30 at 09-43-46 Flax](https://user-images.githubusercontent.com/5862228/176707125-c6f987f2-f1d4-469c-8683-919a7d34e729.png)

I wont remove it immediately but I am aware this won't go through.

Reply:
Not per se against a dramatic entrance ;)

Note also that FastAPI has a bit of a different setup on the page - their actual logo (circle with flash) is much smaller, and they have large text next to it. (not saying we should do that, but if they simply had the huge green circle with flash logo I wouldn't like that too much)